kimchi/wasm/src/pasta_fp_plonk_index.rs

```diff
@@ -32,55 +32,53 @@ pub fn caml_pasta_fp_plonk_index_create(
     public_: i32,
     prev_challenges: i32,
     srs: &WasmSrs,
-) -> Result<WasmPastaFpPlonkIndex, JsValue> {
+) -> Result<WasmPastaFpPlonkIndex, JsError> {
```

Comment:
improve errors from Rust. this used to throw a string (`throw "something"`) in JS, now throws a proper JS error with stack trace